intro_to_rl.ipynb

```diff
@@ -0,0 +1,3225 @@
+{
```

Comment:
<ol><li>Lots of inconsistencies with JAX versus jax. Also JAX Numpy should be JAX NumPy, but prefer <code>jax.numpy.where</code> or <code>jax.lax.select</code>.</li><li>I also think introducing <code>jax.lax.select</code> here is cryptic, as the doc page you are linking to only tells you that it "Wraps XLA’s Select operator." </li><li>stocastic -&gt; stochastic.</li><li>Usefull -&gt; Useful</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>fixed this</p>